sdk/quantum/ci.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
```

Comment:
The indentation appears to be wrong here, it is likely this yml won't be parsed correctly.

Reply:
Fixed the indentation